reference/lifecycle-terminology.md

```diff
@@ -0,0 +1,61 @@
+Software Lifecycle Terminology
+==============================
+
+We work on software that's anywhere in its lifecycle - from something that we've created just to see if it's possible, through to widely-used and well-maintained software that's used in production. 
+
+In order to be clear about what people should expect from a piece of software, we use specific terminology. This page documents our interpretation of these terms, and is intended to be referred to by developers at ODSC, developers that we're collaborating with, and clients. 
+
+Any particular piece of software might have elements that are at different stages of development. For example, a new feature in a mature software product might be a prototype, while the main software product is production-ready. 
+
+As a rule of thumb, it's approximately an order of magnitude of extra work, complexity or cost to move up each level; that is to say that a production implementation of a feature is 100x more complex than its prototype. 
+
```

Comment:
You can bring that 100x more complex down if you have a 10x engineer though right ;)